src/software/ai/hl/stp/tactic/crease_defender/crease_defender_fsm.cpp

```diff
@@ -125,5 +130,12 @@ std::optional<Point> CreaseDefenderFSM::findDefenseAreaIntersection(
             return right_intersections[0];
         }
     }
+    // Check to see if ray is within goalie box to ignore
+    if (contains(inflated_defense_area, ray.getStart()))
+    {
+        // Returns in center front of box if the ball is already in the goalie box
+        return Point(inflated_defense_area.posXPosYCorner().x(), 0);
```

Comment:
yikes if we have multiple crease defenders, this might cause all of our robots to crash into each other

Reply:
check the `crease_defender_alignment` parameter and offset the position here from the center of the friendly crease

Reply:
![Screenshot 2024-06-15 13:50:19](https://github.com/UBC-Thunderbots/Software/assets/42703774/74c38ea4-dbed-4bf8-ab2d-9d1364da377d)